x-pack/plugins/security_solution/server/lib/detection_engine/rule_execution_log/rule_registry_log_client/rule_registry_log_client.ts

```diff
@@ -228,7 +228,7 @@ export class RuleRegistryLogClient implements IRuleRegistryLogClient {
         [getMetricField(metric)]: value,
         [RULE_ID]: ruleId,
         [TIMESTAMP]: new Date().toISOString(),
-        [ALERT_OWNER]: 'siem',
+        [ALERT_CONSUMER]: 'siem',
```

Comment:
I believe we prefer using `SERVER_APP_ID` instead of a hardcoded `siem`, we could consider updating that now that we are here

Reply:
Updated!